_sass/base/_layout.scss

```diff
@@ -38,10 +39,65 @@ header {
 		text-transform: lowercase;
 	}
 
+	.dropdown {
+		a {
+			text-align: left;
+			display: block;
+		}
+		.arrow::after  {
+			font-family: "FontAwesome";
+			content: "\f078";
+			padding: 2px 2px;
+			font-size: 1rem;
+		}
+		&.open {
+			a {
+				color: white;
+				background-color: $brand-color;
+				padding: 8px;
```

Comment:
(funny now, commenting myself.. kind of schizophrenic) I just realized that this line is breaking the alignment between the parent links in the menu. So I have to figure out a way of changing this padding-style change without breaking the original alignment....  Anyone has any ideas?